features/steps/thens/alignment.py

```diff
@@ -61,15 +61,20 @@ def count_segments(logic, representation):
     and geometry representation.
     """
     try:
-        logic_count = len(logic.segments)
+        expected_count = 0
+        for seg in logic.segments:
+            if seg.PredefinedType == "HELMERTCURVE":
+                expected_count += 2
+            else:
+                expected_count += 1
     except AttributeError:
-        logic_count = None
+        expected_count = None
     try:
         rep_count = len(representation.segments)
     except AttributeError:
         rep_count = None
 
-    return logic_count, rep_count
+    return expected_count, rep_count
```

Comment:
I'd say be consistent with using `expected_count` instead of `logic_count`.

I don't seem to be able to pinpoint the exact line numbers in pull requests. But for example in line 164/169 we still say 
`if logic_count != rep_count:`

This should then be 
`if expected_count != rep_count`

Reply:
Renamed as suggested.